MSR - Rename Surface Rooms

dyceron opened the pull request:
Proposed room renames for Surface East. Will fix tests if names are good.

Morph Ball Room -> Morph Ball Chamber
Charge Beam Room -> Charge Beam Chamber
Scan Pulse Room -> Scan Pulse Chamber
Side Path -> Moheek Connector
One-Way Drop -> Upper Storage Access
Charge Beam Shaft -> Charge Beam Chamber Access

Miepee commented:
> Moheek Connector

Not a fan of the word "connector". I'd rather replace with something else, bonus points if it's an alliteration. Moheek Market?

Some rooms in surface east that should also get renamed IMO:
- Entrance pathway
- melee tutorial pathway
- charge beam storage (gets confusing with charge beam chamber)
- Alpha Arena Access (it's so far away from the actual arena, that i dont consider it "access")
- Alpha Storage
- Crumble Drop Shaft
- Landing Site Storage? With the mini puzzle in there and the spikes+crumble blocks everywhere it doesn't really fit like a "storage".
- Am unsure about "upper storage". "Upper" is very relative and if you look at it from the whole area, then it's not really that much up. It's only up in its own scan pulse subsection

In Addition, i'd like for the following rooms to get split:
- Alpha arena should be split into two: one for the arena, one for the big room with the save station before it
- Chozo seal should be split into two: one for the upper part, one for the mini room with the missile tank
- Scan Pulse Chamber should be split into two: one for the actual chamber, one for the right part.

Miepee commented:
> Alpha Arena also doesn't really make sense to split

It does. Currently ~70% of the room "Alpha Arena" is *not* an arena which thus sounds very misleading.
Scan Pulse Chamber has that too, but to a lesser extent. But you're right, that I completely forgot to look for this one on whether this could be sensibly split.
Alpha Arena I would just split before the missile block:
![grafik](https://github.com/randovania/randovania/assets/38186597/de621e99-5027-4633-aa03-2e941e996cbf)

dyceron commented:
I've changed more names and also added two new rooms.

- Alpha Arena Access -> Energy Recharge Station Shaft
- Moheek Connector -> Moheek Meeting
- Entrance Pathway -> Hornoad Hallway
- Melee Tutorial Pathway -> Winding Path
- Charge Beam Storage -> Secret Stash
- Alpha Storage -> Cache West
- Upper Storage Access -> Split Path
- Upper Storage -> Split Path Storage
- Landing Site Storage -> Landing Site Puzzle
- Crumble Drop Shaft -> Landing Site Funnel
- Transport Storage -> Cache East

Split Alpha Arena into two

![image](https://github.com/randovania/randovania/assets/38679103/3c124cc2-860e-4156-be28-27013118e948)
![image](https://github.com/randovania/randovania/assets/38679103/45f25cb8-0a6b-4092-bac5-be1618c720b8)

Added Gravitt Grotto (Split from Chozo Seal)

![image](https://github.com/randovania/randovania/assets/38679103/57883779-9598-4c4b-a4d6-82686a31fc4f)
![image](https://github.com/randovania/randovania/assets/38679103/ef23735f-2966-45d7-9fe7-b7c9a24fb889)
